src/instructions/VerifiedInstructions.jsx

```diff
@@ -9,10 +9,12 @@ import Footer from './proctored_exam/Footer';
 const VerifiedExamInstructions = ({ examType }) => {
   const renderInstructions = () => {
     switch (examType) {
-      case ExamType.ONBOARDING:
-        return <VerifiedOnboardingExamInstructions />;
       case ExamType.PROCTORED:
         return <VerifiedProctoredExamInstructions />;
+      case ExamType.PRACTICE:
+        return <VerifiedProctoredExamInstructions />;
```

Comment:
Nit: Just for clarity, this could be written as:
```suggestion
      case ExamType.PROCTORED:
      case ExamType.PRACTICE:
        return <VerifiedProctoredExamInstructions />;
```